src/Microsoft.Health.Dicom.Core/Features/Query/Model/SeriesResult.cs

```diff
@@ -41,4 +42,14 @@ public DicomDataset DicomDataset
             return _dicomDataset;
         }
     }
+
+    public static readonly IReadOnlyCollection<DicomTag> AvailableTags = new HashSet<DicomTag>()
+    {
```

Comment:
Instead of having this here, should we consolidate in QueryLimit.cs?

Reply:
These are the tags available from StudyResult Stored procedure.  Things in the QueryLimit are more like constants